serverless.yml

```diff
@@ -62,6 +88,12 @@ functions:
     image:
       name: zip
     name: IngestZip-${opt:stage, self:provider.stage, 'dev'}
```

Comment:
@ingalls I'm a little confused here. This appears to be defining `IngestZip` as a Lambda, but my understanding is that the code is never called as such, it's invoked in the AWS Batch Compute Environment (Fargate) as specified lower down in this file in the `BatchJobDefinition` Resource. I think this also may explain my confusion as to why when I look at the logs for the `IngestZip` lambda there's nothing in there. Do we need this?

Reply:
got it. Ok makes sense!